### PR TITLE
Rails 6.1 Support

### PR DESCRIPTION
**Pull request by corasaurus-hex**
Tests pass locally, Circle CI can be ignored for now because it hasn't been set up since the security incident.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -85,13 +83,12 @@ GEM
...
     thor (1.2.1)
-    thread_safe (0.3.6)
-    tzinfo (1.2.10)
-      thread_safe (~> 0.1)
+    tzinfo (2.0.5)
+      concurrent-ruby (~> 1.0)
     zeitwerk (2.6.6)
 
 PLATFORMS
-  ruby
```
Comment (bigSiebs) [inline Gemfile.lock:94]: I'm surprised to see this change?

**Review by bigSiebs — COMMENTED** [review]
Overall, the changes look expected, other than the change of the platform in the Gemfile. Any idea why that changed?

File: Gemfile.lock (hunk 1 above)
Comment (corasaurus-hex) [reply Gemfile.lock:94]: Hmmm yeah, I wonder what that's about. I'll change that to be both ruby and the platform bundler wants it to be.

Comment (corasaurus-hex) [reply Gemfile.lock:94]: Even more strange, it changes the Gemfile.lock to also support the `ruby` platform: https://github.com/wunderteam/globalid-utils/pull/19/commits/ef28feed1249c7f81961926014a406200f6079eb

I'm not sure what it's about but tests still run so ¯\\_(ツ)_/¯

**Pull request merged**